packages/edit-post/src/classic.scss

```
@@ -2,7 +2,7 @@
 // Specificity is kept at this level as many classic themes output
 // rules like figure { margin: 0; } which would break centering.
 // These rules should also not apply to direct children of flex layout blocks.
-.editor-styles-wrapper :where(:not(.is-layout-flex, .is-layout-grid)) > .wp-block {
+.editor-styles-wrapper :where(:not(.is-layout-flex, .is-layout-grid)) > .wp-block:not(.wp-block-list .wp-block-list) {
```

#60802 raised the specificity to the previous level of `0-2-0`, but I do not think it should increase any more.

```suggestion
.editor-styles-wrapper :where(:not(.is-layout-flex, .is-layout-grid)) > .wp-block:where(:not(.wp-block-list .wp-block-list)) {
```

---

Thanks for the feedback, Have updated the specificity.